### PR TITLE
feat: slots minigame

### LINKED ISSUE #496: Create slots minigame

### PR DESCRIPTION
**Pull request by joaodiaslobo**
Depends on safira [#372](https://github.com/cesium/safira/pull/372), resolves #496 
image
image

### CODE REVIEW

File: layout/Attendee/Slots/components/Machine/index.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+// Time to wait before sending the response after the reels stop
+const extraTime = 1000;
+
+// Number of icons in each reel
+const numIcons = slots.symbols;
+
+// Size of the icons
+var iconSize = 79;
+
+interface MachineProps {}
```
Comment (ruilopesm) [inline layout/Attendee/Slots/components/Machine/index.tsx:16]:
```suggestion
```

File: layout/Attendee/Slots/components/SlotsMessage/index.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { Dialog } from "@headlessui/react";
+import { useRouter } from "next/router";
+
+export default function SlotsMessage({ title, description, onExit }) {
+  const router = useRouter();
```
Comment (ruilopesm) [inline layout/Attendee/Slots/components/SlotsMessage/index.tsx:5]: What is this used for?

**Comment by joaodiaslobo** [comment]
[feat: slots minigame](https://trello.com/c/wwSRzl1n/80-feat-slots-minigame)

**Review by MarioRodrigues10 — APPROVED** [review]
What an awesome job you did here :blue_heart: :robot:

**Pull request merged**